docs/example.ts

```diff
@@ -27,10 +42,20 @@ async function setup(): Promise<{
 
   // To get an attestation, we need an Attester
   // we can generate a new keypair:
-  // const attester = await Kilt.AttesterIdentity.buildFromMnemonic("...")
+
+  const keyring = new Kilt.Utils.Keyring({
+    // KILT has registered the ss58 prefix 38
+    ss58Format: 38,
+    type: 'ed25519',
```

Comment:
Should we use `sr25519`?

Reply:
No, since it is running off the CI pipeline which requires the faucet to be `ed25519` for the standalone verison.